VCFv4.3.tex

```diff
@@ -171,30 +171,58 @@ \subsubsection{Individual format field format}
+  \item INS Insertion relative to the reference
   \item DUP Region of elevated copy number relative to the reference
   \item INV Inversion of reference sequence
   \item CNV Copy number variable region (may be both deletion and duplication)
   \item BND Breakend
 \end{itemize}
 
-The CNV category should not be used when a more specific category can be applied. Reserved subtypes include:
-\begin{itemize}
-  \item DUP:TANDEM Tandem duplication
-  \item DEL:ME Deletion of mobile element relative to the reference
-  \item INS:ME Insertion of a mobile element relative to the reference
-\end{itemize}
+The list of first level symbolic alleles that represent structural variants is a closed set.
+Any symbolic alleles with a first level type other than one of those listed above are considered to be non-structural symbolic alleles.
+
+Variant subtypes should be used to add specificity to the maximum extent supported by the data. Reserved subtypes include:
+\newline
+
+\begin{tabular}{l l}
+DEL:ME  &  Absence of mobile element present in the reference \\
+DEL:ME:ALU  &  Absence of Alu element present in the reference \\
+DEL:ME:LINE1  &  Absence of LINE1 element present in the reference \\
+DEL:ME:SVA  &  Absence of SVA element present in the reference \\
+DEL:ME:HERV  &  Absence of HERV element present in the reference \\
+\\
+INS:ME  &  Insertion of mobile element relative to the reference \\
+INS:ME:ALU  &  Insertion of Alu element relative to the reference \\
+INS:ME:LINE1  &  Insertion of LINE1 element relative to the reference \\
+INS:ME:SVA  &  Insertion of SVA element relative to the reference \\
+INS:ME:HERV  &  Insertion of HERV element relative to the reference \\
+\\
+DUP:TANDEM  &  Tandem duplication, in same orientation as original \\
+DUP:INV-BEFORE  &  Tandem duplication, inserted before original in inverted orientation \\
+DUP:INV-AFTER  &  Tandem duplication, inserted after original in inverted orientation \\
+\end{tabular}
+\newline
+
+Variants should be written using the most precise type that can be determined by the variant caller.
+For example, if the insertion site of a new copy of a LINE1 element cannot be determined, it would be specified as a DUP of the originating LINE1 element.
+However if the new insertion site can be identified, the variant should be specified as INS:ME:LINE1 at the insertion site.
+
+Note that the DUP type is not restricted to tandem duplications and that the DUP subtypes have different breakpoints than their parent type.
+
+Variant subtypes are an open set and is it valid for a variant caller to define a subtype not specified in the above list. Such subtypes are considered to be instances of the first reserved subtype, with breakpoint and copy number semantics matching the reserved type. For example, INS:ME:PELEMENT is a valid caller-defined subtype, but INV:FOLDBACK is not since foldback inversions do not result in the same breakpoint and copy number changes as a simple inversion.
```

Comment:
I don't think we can make the last part of the paragraph work as part of the specification, because it is an open set with semantics involved. So we can mention what they are considered to be, but not provide any examples.

```suggestion
Variant subtypes are an open set and is it valid for a variant caller to define a subtype not specified in the above list. Such subtypes are considered to be instances of the first reserved subtype, with breakpoint and copy number semantics matching the reserved type.
```

Reply:
The last paragraph is based on examples I've seen in the real world. Why wouldn't we provide examples of what not to do? Would a wording change be sufficient? For example, an explaination of why a caller should not create INV:FOLDBACK or DUP:INVERTED sub-types and should report these events using BND notation.

I definitely want something in the specs that explicitly states that a caller using the above notations will be non-compliant/be calling a different variant than what they think they're calling.